### ISSUE TITLE
Recommendation: Add all elements Selector

### ISSUE THREAD

**Issue by JoshRosenstein** [bupa-digital/shades#3]
https://github.com/bupa-digital/shades/blob/90625bfa0a5f085fd538c0d6bc2ab57de0c0662f/src/style-parser.js#L72

const isSelector = startsWithAny('.', '#', '>','*');

**Comment by MiracleBlue** [comment]
Good catch!  This might be a good one to have a go at yourself if you're up for it :) if you want to try making a fork of the repo, cloning it, and adding this (and a unit test as well) that would be super.

**Comment by MiracleBlue** [comment]
Hmm.  Now that I think about it, I remember the reason I excluded that selector specifically.  It's because given the context, it doesn't make as much sense.  Shades is designed to exist in a document that could be made up of many, many shadow-doms.  In fact, the existence of the `isSelector` check hasn't always been fully justified in my own mind, and I don't think other CSS in JS libraries like Glamorous support this behaviour either.

Unless we find a compelling reason to add the universal selector, lets leave it out for now - and even consider removing the isSelector check all together in future.

**Comment by MiracleBlue** [comment]
Interesting!  To be honest, I think there are some better ways to achieve a tooltip situation, especially in React.  Thank you so much for that example, however.  You've got me thinking a lot more about this.  I'm going to re-open this ticket, but see if I can come up with an approach that you would deem satisfactory, without the need for the special selectors :)

**Comment by JoshRosenstein** [comment]
Yesterday I started throwing together a huge repl, to keep all my explorations and demos in a single place because i already got 47+ sandboxes and i keep trying to search through em to reference things and its just a mess. I'll eventually add a dedicates Shades Tab, that way all my spin ups will be in kept in single spot.
https://k55z4j3rzo.codesandbox.io/roseys-styler/gridStyled
